cli/src/commands/squash.rs

```diff
@@ -157,7 +161,7 @@ impl SquashedDescription {
         if !args.message_paragraphs.is_empty() {
             let desc = join_message_paragraphs(&args.message_paragraphs);
             SquashedDescription::Exact(desc)
-        } else if args.use_destination_message {
+        } else if args.use_destination_message || args.keep_empty {
```

Comment:
nit: perhaps, this change is redundant. `.abandon` wouldn't be set if `!keep_empty`, and `Combine` means "combine abandoned".